### PR TITLE
Saving data in db

### PR DESCRIPTION
**Pull request by lucianoeli**

### CODE REVIEW

**Review by SartiMau — APPROVED** [review]

File: data/src/main/java/com/globant/data/repositories/MarvelCharacterRepositoryImpl.kt [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
-            val realm = Realm.getDefaultInstance()
-            val character = realm.where(MarvelCharacterRealm::class.java).equalTo("id", id).findFirst()
-            character?.let {return Result.Success(mapper.transform(character)) }
-            return Result.Failure(Exception("Character not found"))
+            return CharacterDatabase.getCharacterById(id)
+        }
+    }
+
+    private fun insertOrUpdateCharacter(character: MarvelCharacter) {
+        val realm = Realm.getDefaultInstance()
```
Comment (capraber) [inline data/src/main/java/com/globant/data/repositories/MarvelCharacterRepositoryImpl.kt]: Why you use a val if you don't do anything with it?

File: data/src/main/java/com/globant/data/service/CharacterService.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+class CharacterService {
+
+    companion object {
+
+        private val api: MarvelResquestGenerator = MarvelResquestGenerator()
+        private val mapper: CharacterMapperService = CharacterMapperService()
+
+        fun getCharacterById(id: Int): Result<MarvelCharacter> {
+            val callResponse = api.createService(MarvelApi::class.java).getCharacterById(id)
+            val response = callResponse.execute()
```
Comment (capraber) [inline data/src/main/java/com/globant/data/service/CharacterService.kt]: add NPE to call Response

**Review by priettt — APPROVED** [review]

File: data/src/main/java/com/globant/data/service/CharacterService.kt (hunk 2 above)
Comment (SartiMau) [reply data/src/main/java/com/globant/data/service/CharacterService.kt]: fixed in next PR

Comment (SartiMau) [reply data/src/main/java/com/globant/data/service/CharacterService.kt]: done

**Pull request merged**